webextensions/api/proxy.json

```diff
@@ -101,6 +101,216 @@
                 "version_added": false
               }
             }
+          },
```

Comment:
@bershanskiy  I believe you may have added the wrong set of properties here. `thirdParty` was added to  `proxy.RequestDetails` not to `proxy.ProxyInfo`, whose properties you appear to have added here. See [proxy.json ](https://hg.mozilla.org/integration/mozilla-inbound/diff/26616eb7c1749dcbcfd601966da08c6a48db46e5/toolkit/components/extensions/schemas/proxy.json)

Reply:
@rebloor I'm confused, I believe I added info to `proxy.RequestDetails`. As a sanity check, I did the following in Node CLI (copied verbatim):
```
$ node
> const api = require('./webextensions/api/proxy')
undefined
> typeof api.webextensions.api.proxy.ProxyInfo.thirdParty
'undefined'
> typeof api.webextensions.api.proxy.RequestDetails.thirdParty
'object'
```

As you see, I added `thirdParty` to `api.webextensions.api.proxy.RequestDetails.thirdParty` and not to `api.webextensions.api.proxy.ProxyInfo`.

Could you take another look?

Reply:
Yeah, this was added to `RequestDetails`, so this is fine!